### ISSUE TITLE
[Research] Engaging Park Operators

### ISSUE THREAD

**Issue by LindsayMacfarlane** [bcgov/reserve-rec-admin#18]
#### Description:
Reach out to Park Operators to learn more about their current experiences - what's working, what isn't, and opportunities for improvement. We will need to be mindful in how we approach POs and how we frame our messaging. 

Note: PO sweet spot for engagement is March. 

#### Acceptance Criteria:
[Note: Use 'Given/When/Then' format if it makes sense to. Otherwise, a simple checklist that can be tested.]

#### Development Checklist:

- [ ] Determine POs that we can contact
- [ ] Determine approach for engagement
- [ ] Work with Reservation Services Team (Carla) on communication approach

#### Dependencies

- Blocked by
- Blocking

**Relevant documentation as reference**
- https://bcparksdigital.atlassian.net/jira/software/projects/RIDA/boards/4?selectedIssue=RIDA-145

**Definition of Ready**

- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if applicable)
- [ ] Design / Solution is accepted by Product Owner (if applicable)
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** 
- In progress:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] UI meets accessibility requirements
  - [ ] Unit tests are written
  - [ ] Work is traceable in GitHub
  - [ ] PR linked to ticket number
  - [ ] If needed/required - Dev adds flag/label to highlight any migration steps necessary prior to PROD deployment
- Code review:
  - [ ] Code is peer reviewed and has passed CI/CD tests
- QA:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Code is potentially shippable to the production environment
  - [ ] Functional features have been tested and passed by QA
  - [ ] UI components tested by designer
  - [ ] Code is deployed to PROD when moved to 'done' column (unless requested otherwise by PO)
- PO Review:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Reviewed and approved by Product Owner

#### Notes:
- [List of POs to start with engagement](https://bcgov.sharepoint.com/:x:/r/teams/08548/Shared%20Documents/General/1.%20Project%20Documentation/4.%20Parks%20%26%20Rec%20Digital%20Transformation%20Project/Park%20Operators/po%20contact.xlsx?d=w344f70523a834d5c80d6679e7623830b&csf=1&web=1&e=jFCVNP). 
-

**Comment by LindsayMacfarlane** [comment]
This ticket may now be a duplicate. Need to confirm.

**Comment by Dianadec** [comment]
Closing as duplicate of 21.
